[2022-10-22 11:05:02,108] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: target_ingest_dag.source_ingest_sensor scheduled__2022-10-22T10:05:00+00:00 [queued]>
[2022-10-22 11:05:02,127] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: target_ingest_dag.source_ingest_sensor scheduled__2022-10-22T10:05:00+00:00 [queued]>
[2022-10-22 11:05:02,128] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-22 11:05:02,130] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-22 11:05:02,131] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-22 11:05:02,149] {taskinstance.py:1377} INFO - Executing <Task(ExternalTaskSensor): source_ingest_sensor> on 2022-10-22 10:05:00+00:00
[2022-10-22 11:05:02,156] {standard_task_runner.py:52} INFO - Started process 1106 to run task
[2022-10-22 11:05:02,160] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'target_ingest_dag', 'source_ingest_sensor', 'scheduled__2022-10-22T10:05:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/target_ingest_dag.py', '--cfg-path', '/tmp/tmpml31knn7', '--error-file', '/tmp/tmpapeum3xg']
[2022-10-22 11:05:02,162] {standard_task_runner.py:80} INFO - Job 9: Subtask source_ingest_sensor
[2022-10-22 11:05:02,257] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/dagrun.py:485 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-10-22 11:05:02,271] {task_command.py:369} INFO - Running <TaskInstance: target_ingest_dag.source_ingest_sensor scheduled__2022-10-22T10:05:00+00:00 [running]> on host 865a121acdee
[2022-10-22 11:05:02,318] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/dagrun.py:485 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-10-22 11:05:02,400] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/renderedtifields.py:213 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-10-22 11:05:02,415] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=rongshiu@gmail.com
AIRFLOW_CTX_DAG_OWNER=rongshiu@gmail.com
AIRFLOW_CTX_DAG_ID=target_ingest_dag
AIRFLOW_CTX_TASK_ID=source_ingest_sensor
AIRFLOW_CTX_EXECUTION_DATE=2022-10-22T10:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-22T10:05:00+00:00
[2022-10-22 11:05:02,417] {external_task.py:166} INFO - Poking for tasks ['source_ingestion_task'] in dag source_ingest_dag on 2022-10-22T10:00:00+00:00 ... 
[2022-10-22 11:05:02,435] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/sensors/external_task.py:230 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-10-22 11:06:02,380] {external_task.py:166} INFO - Poking for tasks ['source_ingestion_task'] in dag source_ingest_dag on 2022-10-22T10:00:00+00:00 ... 
[2022-10-22 11:06:02,407] {base.py:301} INFO - Success criteria met. Exiting.
[2022-10-22 11:06:02,435] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=target_ingest_dag, task_id=source_ingest_sensor, execution_date=20221022T100500, start_date=20221022T110502, end_date=20221022T110602
[2022-10-22 11:06:02,473] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-22 11:06:02,537] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/taskinstance.py:2445 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-10-22 11:06:02,571] {local_task_job.py:273} INFO - 4 downstream tasks scheduled from follow-on schedule check
